### ISSUE TITLE
dependency parser doesn't deal with json-syntax role definitions

### ISSUE THREAD

**Issue by joaocc** [rtuin/ansible-install-dependencies#2]
For instance, consider the following "meta/main.yml" file:
```` 
...
dependencies:
  - my-role-1
  - { role: my-role-2 }
  - { role: my-role-3, my_role_var_ccc: 333 }
....
````
All role specs except "my-role-1" cause the script to crash.
Any chance of finding if the element is a {} and extracting the "role:" component?
Thx

**Comment by rtuin** [comment]
Great idea, thanks for the fix in #3. I'll test it asap and hope to merge it :)

**Comment by rtuin** [comment]
Closed when merging #3
